packages/firebase_ml_vision/example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -38,6 +38,7 @@
 /* End PBXCopyFilesBuildPhase section */
```

Comment:
You can remove this file from being checked in.

Reply:
removing this file fails the CI, saying that the application isn't configured for iOS, I've added it back for now